src/js/controller/tracks-loader.ts

```diff
@@ -64,14 +74,15 @@ function xhrSuccess(xhr, track, successHandler, errorHandler) {
             const responseText = xhr.responseText;
             if (responseText.indexOf('WEBVTT') >= 0) {
                 // make VTTCues from VTT track
-                loadVttParser().then(VTTParser => {
+                loadVttParser().then((VTTParser: (window: Window, decoder?: () => GenericObject) => void): void => {
```

Comment:
`VTTParser` typing should be temporary until vttparser.js is TSed. Can you put that in types/generic.type for now or below in `loadVttParser(): Promise<(window: Window, decoder?: () => void)>`? It doesn't return an object, we get one with the `new` keyword.